### PR TITLE
Maintenance: batch of minor changes

### PR DESCRIPTION
**Pull request by bzz**
Small cleanup and simplification of Makefile, .gitignore, doc and tests

### CODE REVIEW

File: benchmarks/parse.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
-#!/bin/sh
-
-cd benchmarks/output && go run ../parser/main.go -outdir ../csv && \
-cd ../csv && go run ../parser/main.go -distribution
+#!/usr/bin/env bash
```
Comment (creachadair) [inline benchmarks/parse.sh]:
```suggestion
 set -e
```

Comment (creachadair) [reply benchmarks/parse.sh]: and so on for the other scripts that have sequences of actions that depend on each other.

**Comment by bzz** [comment]
@creachadair thank you for prompt feedback. Addressed and ready for another round.

File: Makefile [hunk 2]
```diff
@@ -54,17 +48,17 @@ code-generate: $(LINGUIST_PATH)
...
+	go test -run=NONE -bench=. -benchtime=5us
+	benchmarks/linguist-samples.rb
 
 benchmarks-slow: $(LINGUST_PATH)
-	mkdir -p benchmarks/output && go test -run=NONE -bench=. -slow -benchtime=100ms -timeout=100h >benchmarks/output/enry_samples.bench && \
-	benchmarks/linguist-samples.rb 5 >benchmarks/output/linguist_samples.bench
-
-build-cli:
-	go build -o enry -ldflags "$(LOCAL_LDFLAGS)" cmd/enry/main.go
+	mkdir -p benchmarks/output
```
[the remark this answers is not in the document]
Comment (bzz) [reply Makefile]: Reverted, as this is a Makefile

**Pull request merged**